docs/default.nix

```
@@ -72,6 +72,7 @@ in
       nuschtosSearch.packages.mkSearch {
         optionsJSON = options-json + "/share/doc/nixos/options.json";
         urlPrefix = "https://github.com/nix-community/nixvim/tree/main";
+        title = "NixVim Options Search";
```

```suggestion
        title = "Nixvim options search";
```

Maybe this is more natural. I have no strong feeling though.